src/sdl3/iostream.rs

```diff
@@ -12,6 +12,8 @@ use crate::sys;
 /// A structure that provides an abstract interface to stream I/O.
 pub struct IOStream<'a> {
     raw: NonNull<sys::iostream::SDL_IOStream>,
+    #[allow(dead_code)]
```

Comment:
what about putting buf before raw, so that raw is dropped before buf is freed. when drop is called it calls `SDL_CloseIO(raw)` which might depend on the memory allocated in buf.

Reply:
Aren't fields dropped in order? (ref: https://stackoverflow.com/questions/41053542/forcing-the-order-in-which-struct-fields-are-dropped )
I believe raw is already being dropped before buf.